documentation/user-consent.md

```diff
@@ -0,0 +1,214 @@
+# User Consent Management
+
+This document captures guidelines for Operator NaaS platform to handle user consents.
+
```

Comment:
Outlining what managing user consent means would be helpful.
Some points that can be considered
- Select an identifier which can be used to create the user consent records
- Allow the operator to create, read, update, and delete consent data records 
- Validate an API request against a policy enforcement point to check if consent already exists or not
- Allow users to view and update their consents 
- Describe a flow how consent can be collected from the user without degrading the user experience while using a third-party service

Reply:
I included an explanation in introduction chapter, in commit https://github.com/camaraproject/IdentityAndConsentManagement/pull/13/commits/c7b23f53b47ba92afa9eab8aeb36083962b76cfe. The document will then get into details, in different PRs as this one is intended to set the baseline.

I included your suggestions with some rewording, please check.

Reply:
@shilpa-padgaonkar Changes included are ok with you?

Reply:
On behalf of @shilpa-padgaonkar: yes, changes ok.